inst/adducts/adduct_definition.txt

```diff
@@ -0,0 +1,50 @@
+"name"	"mass_multi"	"mass_add"	"formula_add"	"formula_sub"	"charge"	"positive"
```

Comment:
This file was created using the original adduct definitions (within "adducts.R").